src/main/java/step3/domain/Winning.java

```diff
@@ -0,0 +1,63 @@
+package step3.domain;
+
+import java.util.Arrays;
+
+public enum Winning {
+    NOTHING(0, 0),
+    FIFTH(3, 5000),
+    FOURTH(4, 50000),
+    THIRD(5, 1500000),
+    SECOND(5, 30000000),
+    FIRST(6, 2000000000);
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000);
```

언더바를 통해 가독성을 향상시킬 수 있습니다!